tests/e2e/e2e_trafficsplit_same_sa_test.go

```diff
@@ -53,7 +53,6 @@ var _ = Describe("Test TrafficSplit where each backend shares the same ServiceAc
 		It("Tests HTTP traffic from Clients to the traffic split Cluster IP", func() {
 			// Install OSM
 			Expect(td.InstallOSM(td.GetOSMInstallOpts())).To(Succeed())
-			Expect(td.WaitForPodsRunningReady(td.osmNamespace, 90*time.Second, 1)).To(Succeed())
```

Comment:
Instead of removing this can we have a check to see there's a pod and its in running state ? 

Applies to all places

Reply:
check `common.go`, patch adds it at the end of `InstallOSM` as it was called basically everywhere